docs/CHANGES.rst

```diff
@@ -0,0 +1 @@
+../CHANGES.rst
```

Comment:
(I think putting the title actually in index.rst might mess up Sphinx's navigation structure)